ida_db/ida_db.py

```diff
@@ -97,9 +97,7 @@ def log(self,table,channels,time=0):
         if type(channels)==str:
             channels_string = channels
         else:
-            import numpy
-            channels_string = numpy.array2string(channels,separator=',')
-            channels_string = channels_string[1:-1] # remove brackets
+            channels_string = ','.join(map(str, channels))

```

Comment:
Note that while `array2string` leaves the resulting string with square brackets at the start and end, `join` does not.